Destination/StreamDestination.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Zenstruck\BackupBundle\Destination;
 
+use ArrayIterator;
```

Comment:
Ok.

> On Sep 11, 2015, at 3:32 PM, Kevin Bond <notifications@github.com> wrote:
> 
> In Destination/StreamDestination.php <https://github.com/kbond/ZenstruckBackupBundle/pull/3#discussion_r39270812>:
> 
> > @@ -2,6 +2,7 @@
> >  
> >  namespace Zenstruck\BackupBundle\Destination;
> >  
> > +use ArrayIterator;
> For global namespaces, can you just inline them instead of adding the use.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/kbond/ZenstruckBackupBundle/pull/3/files#r39270812>.
>